app/javascript/packages/document-capture/components/in-person-location-step.tsx

```diff
@@ -30,9 +30,62 @@ interface FormattedLocation {
   weekdayHours: string;
 }
 
+interface PostOptions {
+  /**
+   * Whether to send the request as a JSON request. Defaults to inclusion.
+   */
+  json: boolean;
+
+  /**
+   * Whether to include CSRF token in the request. Defaults to inclusion.
+   */
+  csrf: boolean;
+
+  /**
+   * Optional. HTTP verb used. If excluded, will fetch as a GET.
+   */
+  method: string;
```

Comment:
If method is an option and it defaults to GET, should the options name be something other than "PostOptions" ? To me, "Post" would imply a POST request.

Maybe `RequestOptions` ?